Testing 1/3 : Teste 1
Testing 2/3 : Teste 2
Testing 3/3 : Teste 3

<|--
-Failed tests
Test 1: Teste 1
Test 2: Teste 2
Test 3: Teste 3
--|>

<|--
-Test 1: Teste 1 (-3.333)
Incorrect program result
 --- Input ---
> 2
>4
>3
>2

 --- Program output ---
>digite o nome de T1:digite o nome de T2:digite o nome de T3:digite o nome de T4:Traceback (most recent call last):
>  File "tomadas.py", line 11, in <module>
>    print('%d' %t)
>NameError: name 't' is not defined

 --- Expected output (numbers)---
>8

-Test 2: Teste 2 (-3.333)
Incorrect program result
 --- Input ---
> 6
>6
>6
>6

 --- Program output ---
>digite o nome de T1:digite o nome de T2:digite o nome de T3:digite o nome de T4:Traceback (most recent call last):
>  File "tomadas.py", line 11, in <module>
>    print('%d' %t)
>NameError: name 't' is not defined

 --- Expected output (numbers)---
>21

-Test 3: Teste 3 (-3.333)
Incorrect program result
 --- Input ---
> 2,2,2,2

 --- Program output ---
>digite o nome de T1:digite o nome de T2:Traceback (most recent call last):
>  File "tomadas.py", line 7, in <module>
>    T2=input('digite o nome de T2:')
>EOFError: EOF when reading a line

 --- Expected output (numbers)---
>5

--|>

<|--
-Summary of tests
>+------------------------------+
>|  3 tests run/ 0 tests passed |
>+------------------------------+

--|>

Grade :=>> 0
